msbuild/Xamarin.iOS.Tasks.Core/Tasks/MTouchTaskBase.cs

```diff
@@ -681,7 +684,30 @@ public override bool Execute ()
 
 			Directory.CreateDirectory (AppBundleDir);
 
-			return base.Execute ();
+			var result = base.Execute();
+
+			CopiedFrameworks = GetCopiedFrameworks();
+
+			return result;
+		}
+
+		ITaskItem[] GetCopiedFrameworks()
+		{
+			var copiedFrameworks = new List<ITaskItem>();
+			var frameworksDir = Path.Combine(AppBundleDir, "Frameworks");
+
+			if (Directory.Exists(frameworksDir))
+			{
+				foreach (var dir in Directory.EnumerateDirectories(frameworksDir, "*.framework"))
+				{
+					var framework = Path.Combine(dir, Path.GetFileNameWithoutExtension(dir));
+
+					if (File.Exists(framework))
+						copiedFrameworks.Add(new TaskItem(framework));
+				}
+			}
+
+			return copiedFrameworks.ToArray();
```

Comment:
Style: missing space before (

Reply:
The style in msbuild also uses `{` on the same line as the foreach/if statements.